resources/Components/KeyboardShortcutConfigurator/index.js

```diff
@@ -0,0 +1,95 @@
+import React, { useState, useEffect } from 'react'
+import hotkeys from 'hotkeys-js'
+
+import link from '../../../resources/link'
+import { getShortcutFromKeyEvent, getDisplayShortcut, isShortcutKey } from '../../../resources/app'
+
+const KeyboardShortcutConfigurator = ({ actionText = '', platform, shortcut, shortcutName }) => {
+  const [configuring, setConfiguring] = useState(false)
+
+  useEffect(() => {
```

Comment:
can this hook go in the `EnterShortcut` component so we don't need to switch on the state? we can just bind on mount and unbind on unmount